File: doc-examples/example-java/src/main/java/example/Application.java
package example;

import io.micronaut.runtime.Micronaut;
import io.micronaut.serde.annotation.SerdeImport;

@SerdeImport(
    value = Product.class,
    mixin = ProductMixin.class
) // <1>
public class Application {

    public static void main(String[] args) {
        Micronaut.run(Application.class, args);
    }
}

File: doc-examples/example-kotlin/src/main/kotlin/example/Application.kt
package example

import io.micronaut.runtime.Micronaut.*
import io.micronaut.serde.annotation.SerdeImport

fun main(args: Array<String>) {
    build()
        .args(*args)
        .packages("com.example")
        .start()
}

@SerdeImport(
    value = Product::class,
    mixin = ProductMixin::class) // <1>
class Serdes {}